### DISCUSSION TITLE
Looking For a Meaningful Bachelor Thesis Opportunity

### DISCUSSION THREAD

**Discussion by giscus** [VIEWVIEWVIEW/blog#5] [Comments]
# Looking For a Meaningful Bachelor Thesis Opportunity

http://localhost:4321/articles/ba

**Comment by VIEWVIEWVIEW** [comment]
Scraping Project Idea: Analysis of a possible correlation between a company *earnings and employee feedback from Kununu.com feedback.

*Maybe grab that data from the Bundesanzeiger/Handelsregister.
